content/en/docs/next/op-guide/supported-platform.md

```diff
@@ -8,29 +8,29 @@ description: etcd support status for common architectures & operating systems
 
 The following table lists etcd support status for common architectures and operating systems:
 
-| Architecture | Operating System | Status       | Maintainers                 |
+| Architecture | Operating System | Support Tier | Maintainers                 |
 | ------------ | ---------------- | ------------ | --------------------------- |
-| amd64        | Darwin           | Experimental | etcd maintainers            |
-| amd64        | Linux            | Stable       | etcd maintainers            |
-| amd64        | Windows          | Experimental |                             |
-| arm64        | Linux            | Experimental | @glevand                    |
-| arm          | Linux            | Unstable     |                             |
-| 386          | Linux            | Unstable     |                             |
-| ppc64le      | Linux            | Stable       | etcd maintainers, @mkumatag |
+| amd64        | Linux            | Tier-1       | etcd maintainers            |
+| arm64        | Linux            | Tier-2       | @gyuho, @glevand            |
```

Comment:
It may be a good idea to make the GitHub usernames in this table links to those users.